docs/management/api/host-isolation-exceptions-api.asciidoc

```diff
@@ -0,0 +1,469 @@
+	"item_id": "1342081d-66ad-4b03-b554-131195ce404b",
+	"list_id": "endpoint_host_isolation_exceptions",
+	"name": "Host Isolation Exception - Created with API",
+	"namespace_type": "agnostic",
+	"os_types": [
+		"linux",
+		"macos",
+		"windows"
+	],
+	"tags": [
+		"policy:all"
+	],
+	"tie_breaker_id": "b65843e1-07b8-46d2-9e2b-5f0586dc364a",
+	"type": "simple",
+	"updated_at": "2022-03-03T14:04:00.498Z",
+	"updated_by": "elastic"
+}
+--------------------------------------------------
+
+
+
+[#update-item-api]
+==== Update {endpoint-artifact-name-single}
+
+`PUT <kibana host>:<port>/api/exception_lists/items`
+
+===== Request body
+
+A JSON object containing the fields listed below. The request must have the following:
+
+include::_exceptions-api-reusable-content.asciidoc[tags=endpoint-exceptions-request-payload-requirements]
+* `os_types` must be an array with all xref:exception-item-os-types[]
+include::_exceptions-api-reusable-content.asciidoc[tags=update-exception-item-request-body]
+
+
+====== Example request
+
+Updates the `entries`:
+
+[source,console]
+--------------------------------------------------
+PUT api/exception_lists/items
+{
+	"description": "via API",
+	"entries": [
+		{
+			"field": "destination.ip",
+			"type": "match",
+			"value": "10.10.0.15",
+			"operator": "included"
+		}
+	],
+	"id": "c6883700-9afa-11ec-94b1-fd7e90cc2a08",
+	"item_id": "1342081d-66ad-4b03-b554-131195ce404b",
+	"name": "Host Isolation Exception - Created with API",
+	"namespace_type": "agnostic",
+	"os_types": [
+		"linux",
+		"macos",
+		"windows"
+	],
+	"type": "simple"
+}
+--------------------------------------------------
+// KIBANA
+
+
+===== Response code
+
+`200`::
+    Indicates a successful call.
+
+===== Response payload
+
+[source,json,subs="attributes"]
+--------------------------------------------------
+{
+	"_version": "WzIyMDIsMV0=",
+	"created_at": "2022-03-03T14:04:00.496Z",
+	"created_by": "elastic",
+	"description": "via API",
+	"entries": [
+		{
+			"field": "destination.ip",
+			"type": "match",
+			"value": "10.10.0.15",
+			"operator": "included"
+		}
+	],
+	"id": "c6883700-9afa-11ec-94b1-fd7e90cc2a08",
+	"item_id": "1342081d-66ad-4b03-b554-131195ce404b",
+	"list_id": "endpoint_host_isolation_exceptions",
+	"name": "HIE via API",
+	"namespace_type": "agnostic",
+	"os_types": [
+		"linux",
+		"macos",
+		"windows"
+	],
+	"tags": [],
+	"tie_breaker_id": "b65843e1-07b8-46d2-9e2b-5f0586dc364a",
+	"type": "simple",
+	"updated_at": "2022-03-03T14:22:08.345Z",
+	"updated_by": "elastic"
+}
+--------------------------------------------------
+
+
+==== Delete {endpoint-artifact-name-single}
+
+`DELETE <kibana host>:<port>/api/exception_lists/items`
+
+===== URL query parameters
+
+include::_exceptions-api-reusable-content.asciidoc[tags=api-id-or-itemid-query-params]
+
+
+====== Example request
+
+Deletes a {endpoint-artifact-name-single} with `item_id` of `1342081d-66ad-4b03-b554-131195ce404b`:
```

Comment:
Revise to avoid conflict with "a/an" in other artifact name (like "event filter").
```suggestion
Deletes the {endpoint-artifact-name-single} with `item_id` of `1342081d-66ad-4b03-b554-131195ce404b`:
```